phpunit/tests/fonts/font-library/fontFamilyBackwardsCompatibility.php

```diff
@@ -6,18 +6,28 @@
  *
  * @package WordPress
  * @subpackage Font Library
+ *
+ * @group fonts
+ * @group font-library
  */
 class Tests_Font_Family_Backwards_Compatibility extends WP_UnitTestCase {
+	private $post_ids_to_delete = array();
```

Comment:
I can say that this is a redundant initialization of the `post_ids_to_delete` property since it gets initialized in the `set_up()` method anyway.
However, this could also be considered good coding practice to always initialize variables before using them.
Nonetheless, this falls into the category of an ultra-nitpick, and in my opinion, doesn't justify a new pull request. The rest of the code looks good to me.

Reply:
Note that the backwards compatibility code for Font Families [shouldn't go into Core](https://github.com/WordPress/gutenberg/blob/b550b1351220cda16258d1af6d0b5ee7d0720ab7/lib/compat/wordpress-6.5/fonts/fonts.php#L253). This is only to handle the refactor in the latest Gutenberg release, to keep installed fonts working for sites that were already using the Font Library feature from Gutenberg prior to 17.6.

I should have added a `@core-merge` comment in this test, as well, sorry about that.

Reply:
In that case should we break this test into its own file?

Reply:
Yes, I think you're right. As long as `gutenberg_convert_legacy_font_family_format` continues to run for anyone who may have installed fonts previous to Gutenberg 17.6, this should be fine--and it looks like the function will run from this file without issue.

I also think it makes sense to delete this code after WP 6.5 is released--anticipating that almost all who were using the font library before would have upgraded Gutenberg and no longer need the legacy wp_font_family migration.